Gradle.md

```diff
@@ -94,6 +94,15 @@ The scripts return with an exit code that depends on the outcome of running a gi
 | 3         | The build was not fully cacheable for the given task graph and `--fail-if-not-fully-cacheable` was set |
 | 100       | An unclassified, fatal error happened while running the experiment                                     |
 
+## Verifying the setup
+
+You can verify that the script can properly interact with your Develocity server. In the example below, the script will configure 
```

Comment:
```suggestion
You can verify that the script can properly interact with your Develocity server. The example below will configure 
```

Reply:
We decided not to make that change since it would require all places in this doc to be adjusted.